war/src/main/less/modules/page-header.less

```diff
@@ -35,8 +35,6 @@ a.page-header__brand-link {
 .page-header__brand-image {
     height: 32px;
     width: 24px;
-    height: 2rem;
-    width: 1.5rem;
```

Comment:
are you sure the px is the intended one and not the rem? from memory last one wins?

Reply:
Good catch, that should likely be a rem for the object 👍🏻